src/App.scss

```diff
@@ -2,3 +2,75 @@
 @import "~@fortawesome/fontawesome-free/scss/fontawesome.scss";
 @import "~@fortawesome/fontawesome-free/scss/regular.scss";
 @import "~@fortawesome/fontawesome-free/scss/solid.scss";
+
+$pink-one: #fb95ae;
+$blue-one: #70b1d2;
+$pink-two: #f47391;
+$blue-two: #7196d1;
+$yellow: #fec761;
+$black: #0d0809;
```

Comment:
Since we are using bulma, why not override bulma variables?

https://bulma.io/documentation/customize/variables/

This way, the colours would be part of our colour scheme.